apps/wpcom-block-editor/src/wpcom/features/redirect-onboarding-user-after-publishing-post.js

```diff
@@ -9,7 +9,10 @@ export function RedirectOnboardingUserAfterPublishingPost() {
 	const { siteIntent: intent } = useSiteIntent();
 
 	useEffect( () => {
-		if ( intent === START_WRITING_FLOW ) {
+		const hasStartWritingFlowQueryArg =
```

Comment:
```suggestion
		// We check the URL param along with site intent because the param loads faster and prevents element flashing.
		const hasStartWritingFlowQueryArg =
```
Might add a comment here and in the EDK on why we're using the URL parameter still, so it doesn't get removed in the future.

Reply:
I'll do on the 3 places I implemented this strategy